aspnetcore/security/enforcing-ssl.md

```diff
@@ -351,30 +388,79 @@ See [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/6199).
 
 ## Trust HTTPS certificate on Linux
 
-<!-- Instructions to be updated by engineering team after 5.0 RTM. -->
+Establishing trust is browser specific. The following sections provide instructions for the Chromium browsers Edge and Chrome and for Firefox.
+
+### Trust HTTPS certificate on Linux using Edge or Chrome
+
+  * Install the `libnss3-tools` for your distribution.
+  * Create or verify the `$HOME/.pki/nssdb` folder exists on the machine.
+  * Export the certificate with the following command:
+  
+    ```cli
+    dotnet dev-certs https -ep /usr/local/share/ca-certificates/aspnet/https.crt --format PEM
```

Comment:
Worth mentioning that the path here is specific for Ubuntu, but that they can choose a different path or the path for the CA-store on their distro.